src/serializer.cc

```diff
@@ -59,9 +57,14 @@ Model::DeserializeFromFile(FILE* src_fp) {
   auto header_primitive_field_handler = [src_fp](auto* field) {
     ReadScalarFromFile(field, src_fp);
   };
-  DeserializeTemplate(header_primitive_field_handler, threshold_type, leaf_output_type);
+  int32_t major_ver, minor_ver, patch_ver;
```

Comment:
```suggestion
  int32_t major_ver;
  int32_t minor_ver;
  int32_t patch_ver;
```
Similar for any other single-line multi-declarations.